### PR TITLE
ESLint graduation

### PR DESCRIPTION
**Pull request by dmethvin**
Review and comment welcome.

@ilyavolodin

### CODE REVIEW

File: Projects/ESLint.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* [Managing Releases](http://eslint.org/docs/maintainer-guide/issues.html)
+* [Governance](http://eslint.org/docs/maintainer-guide/issues.html)
+
+# Current contribution process
+
+Linked in the README and documented here: http://eslint.org/docs/developer-guide/contributing/
+
+# List of current tools in use by the project (forums, issue trackers, GitHub orgs, etc)
+
+https://github.com/eslint
```
Comment (kborchers) [inline Projects/ESLint.md]: Formatting nit - convert to an actual markdown list so it doesn't just all run together

File: Projects/ESLint.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# Project scope
+
+ESLint helps JavaScript developers write code that avoids common language pitfalls and follows style guidelines. Since ESLint is configurable and extendable, developers and/or development teams have a great deal of flexibility about which rules they want to enforce. This reduces the time spent on style issues during a code review.
+
+# Current governance process --- (decision making process)
+
+A Technical Steering Committee (TSC) determines the direction of the project at a high level and ensures that the project does regular releases, with the current cadence being every two weeks. Meeting notes are kept at https://github.com/eslint/tsc-meetings . The current TSC is described at https://github.com/eslint/eslint#technical-steering-committee-tsc . The procedures for managing projects are as follows:
+* [Managing Issues](http://eslint.org/docs/maintainer-guide/issues.html)
+* [Reviewing Pull Requests](http://eslint.org/docs/maintainer-guide/issues.html)
```
Comment (jlipps) [inline Projects/ESLint.md]: these all link to the same place, i'm assuming they should not be?

Comment (ilyavolodin) [reply Projects/ESLint.md]: Second link should link to https://eslint.org/docs/maintainer-guide/pullrequests.html

File: Projects/ESLint.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* [Managing Releases](http://eslint.org/docs/maintainer-guide/issues.html)
```
Comment (ilyavolodin) [inline Projects/ESLint.md]: https://eslint.org/docs/maintainer-guide/releases.html

File: Projects/ESLint.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* [Governance](http://eslint.org/docs/maintainer-guide/issues.html)
```
Comment (ilyavolodin) [inline Projects/ESLint.md]: https://eslint.org/docs/maintainer-guide/governance.html

File: Projects/ESLint.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# Project description
+
+ESLint is a JavaScript linting tool that audits code and reports problems that it finds. The tool is configurable to accommodate different programming styles and it is also configurable using third-party plugins.
+ 
+# Project history
+
+Nicholas Zakas started the ESLint project in 2015 in order to establish an extensible code auditing framework that used currently-available JavaScript parsers and provided an opportunity for users to create plugins representing personal or team preferences.
```
Comment (not-an-aardvark) [inline Projects/ESLint.md]: I think ESLint was created in 2013, not 2015.

Comment (ilyavolodin) [reply Projects/ESLint.md]: Ah, good catch. That's correct it was started in 2013.

File: Projects/ESLint.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+In April 2016, the JSCS team joined ESLint in a bid to consolidate auditing for both code bugs and code style into a single tool.
+
+The ESLint project is currently a member of the JS Foundation by way of the jQuery Foundation. ESLint joined the jQuery Foundation in April 2016.
+
+# Available metrics or estimates about the user base, ecosystem and community.
+
+ESLint is currently the most popular JavaScript linting tool, and many projects use it as part of their build system. Stats at [https://www.npmjs.com/package/eslint](https://www.npmjs.com/package/eslint) show that it is downloaded more than 300,000 times per day as of August 2017. The project itself is very active; it has more than 530 contributors since its inception and more than 160 contributors in the past year.
+
+On GitHub the main eslint repo has 238 watchers, 8,528 stars, and 1,621 forks. The @geteslint Twitter account has 9,250 followers. ESLint is depended upon by most of the top-starred GitHub projects including Bootstrap, React, d3, Vue, Angular,  jQuery, nodejs, meteor, redux, express, vscode, and webpack.
```
Comment (jlipps) [inline Projects/ESLint.md]: NIT: would be nicer as `The [@geteslint](https://twitter.com/geteslint) Twitter account`

File: Projects/ESLint.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+# Current contribution process
+
+Linked in the README and documented at [https://eslint.org/docs/developer-guide/contributing/ ](https://eslint.org/docs/developer-guide/contributing/)
+
+# List of current tools in use by the project (forums, issue trackers, GitHub orgs, etc)
+
+* [https://github.com/eslint](https://github.com/eslint)
+* Mentioned in [https://github.com/eslint/eslint](https://github.com/eslint/eslint) README:
+  - [http://eslint.org](http://eslint.org)
```
Comment (Aladdin-ADD) [inline Projects/ESLint.md]: https://eslint.org

**Pull request closed without merging**